server.js

```diff
@@ -283,6 +283,21 @@ function handler(from, to, message) {
     }
     return;
   }
+
+  if (message.indexOf('what prs need a reviewer') > -1) {
+    searchGithub('?assignee=none&labels=S-awaiting-review', 'servo', 'servo', function(error, issues) {
+      if (error) {
+        console.log(error);
+        return;
+      }
+
+      issues.forEach(function(issue) {
+        bot.say(to, issue.title + ': ' + issue.url);
+      });
+
+      return;
```

Comment:
Sorry, one last change. This return is redundant inside the callback function, and we should move it to follow the search github call instead.